docs/content/preview/yugabyte-platform/create-deployments/connect-to-universe.md

```diff
@@ -0,0 +1,441 @@
+
+<!--
+
+![YCQL Load Metrics](/images/ee/multi-zone-universe-ycql-load-metrics.png)
+
+-->
+
+You can also view metrics at a per-node level.
+
+<!--
+
+![YCQL Load Metrics Per Node](/images/ee/multi-zone-universe-ycql-load-metrics-per-node.png)
+
+-->
+
+You can stop the load tester as follows:
+
+- Find the container by executing the following command:
+
+  ```shell
+  user@yugaware-1:~$ sudo docker container ls | grep "yugabytedb/yb-sample-apps"
+  ```
+
+  Expect an output similar to the following:
+
+  ```output
+  <container_id> yugabytedb/yb-sample-apps "/usr/bin/java -jar …" 17 seconds ago Up 16 seconds                                                                                                            jovial_morse
+  ```
+
+  For example, if the container ID is ac144a49d57d, you would see the following output:
+
+  ```output
+  ac144a49d57d yugabytedb/yb-sample-apps "/usr/bin/java -jar …" 17 seconds ago Up 16 seconds                                                                                                            jovial_morse
+  ```
+
+- Stop the container by executing the following command:
+
+  ```shell
+  user@yugaware-1:~$ sudo docker container stop <container_id>
+  ```
+
+  Expect the following output:
+
+  ```output
+  <container_id>
+  ```
+
+  For example, for a container with ID ac144a49d57d, you would need to execute the following command:
+
+  ```shell
+  user@yugaware-1:~$ sudo docker container stop ac144a49d57d
+  ```
+
+  You would see the following output:
+
+  ```output
+  ac144a49d57d
+  ```
+
+<!--
+
+### RedisKeyValue workload
+
+To start the RedisKeyValue workload, execute the following command.
+
+```sh
+$ java -jar /home/yugabyte/tserver/java/yb-sample-apps.jar \
+            --workload RedisKeyValue \
+            --nodes $YEDIS_ENDPOINTS \
+            --num_threads_write 2 \
+            --num_threads_read 32 \
+            --value_size 128 \
+            --num_unique_keys 10000000 \
+            --nouuid
+```
+
+The sample application produces output similar to the following and reports some statistics in the steady state:
+
+```output
+Read: 50069.15 ops/sec (0.64 ms/op), 657550 total ops  | Write: 1470.87 ops/sec (1.36 ms/op), 18849 total ops
+Read: 50209.09 ops/sec (0.64 ms/op), 908653 total ops  | Write: 1454.87 ops/sec (1.37 ms/op), 26125 total ops
+Read: 50016.18 ops/sec (0.64 ms/op), 1158794 total ops | Write: 1463.26 ops/sec (1.37 ms/op), 33443 total ops
+```
+
+If you open the **Metrics** tab of the universe, you should see the metrics graphs, as shown in the following illustration:
+
+![YEDIS Load Metrics Per Node](/images/ee/multi-zone-universe-yedis-load-metrics.png)
+
+Note that these server-side metrics tally with the client-side metrics reported by the load tester.
+
+You should stop the sample application.
+
+-->
+
+### Examine data
+
+You can connect to the YCQL service by executing the following command:
```

Comment:
Assuming ysql also works similarly , can you mention about it or include an example?